srcalc/src/main/java/gov/va/med/srcalc/domain/Calculation.java

```diff
@@ -87,13 +88,51 @@ public void setSpecialty(final Specialty specialty)
                     "Cannot return list of variables because no specialty has been set.");
         }
         
-        return fSpecialty.getVariables();
+        return Collections.unmodifiableSet(fSpecialty.getModelVariables());
```

Comment:
Why all of the changes from lists to sets? Is this to deal with larger input data?